<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spoofingProperty</class>
 <widget class="QDialog" name="spoofingProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ipEdit" name="clientEdit" native="true">
     <property name="toolTip">
      <string>Ip address</string>
     </property>
     <property name="whatsThis">
      <string>The field for ip-address.</string>
     </property>
     <property name="labelText" stdset="0">
      <string>Client</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ipEdit" name="serverEdit" native="true">
     <property name="toolTip">
      <string>Ip address</string>
     </property>
     <property name="whatsThis">
      <string>The field for ip-address.</string>
     </property>
     <property name="labelText" stdset="0">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/im/images/ok.png</normaloff>:/im/images/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/im/images/not.png</normaloff>:/im/images/not.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ipEdit</class>
   <extends>QWidget</extends>
   <header>ipedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>spoofingProperty</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
